bob/src/hw_metrics_collector.rs

```diff
@@ -251,3 +428,27 @@ fn bytes_to_mb(bytes: u64) -> u64 {
 fn kb_to_mb(kbs: u64) -> u64 {
     kbs / 1024
 }
+
+fn parse_command_output(output: std::io::Result<Output>) -> Result<String, String> {
+    match output {
+        Ok(output) => match (output.status.success(), String::from_utf8(output.stdout)) {
+            (true, Ok(out)) => Ok(out),
+            (false, _) => {
+                if let Ok(e) = String::from_utf8(output.stderr) {
+                    let error = format!("Command finished with error: {}", e);
```

Comment:
You should add an information about which command was executed (it can be passed as a parameter to function)

Reply:
It should be added to all error messages in this function